tests/lib/wheel.py

```diff
@@ -48,7 +48,7 @@ class Default(Enum):
 Defaulted = Union[Default, T]
 
 
-def ensure_binary(value: AnyStr) -> bytes:
+def ensure_binary(value: Union[bytes, str]) -> bytes:
```

Comment:
I thought `AnyStr` is basically `Union[bytes, str]`. What's the error message for this?

Reply:
AnyStr is defined as `TypeVar('AnyStr', str, bytes)`. Source:
https://docs.python.org/3/library/typing.html#typing.AnyStr

This means the variable can be either str or bytes, but the two can't be mixed. This link, however, allows mixing of byte and str values, so a union is more appropriate.

If these were reverted the error is:

```
tests/lib/__init__.py:1164: error: Cannot infer type argument 1 of "make_wheel"
 [misc]
        wheel_builder = make_wheel(
```

I then proceeded to fix a chain of function calls until mypy stopped emitting errors.

Reply:
Hmm OK.